features/step_definitions/record_of_guidance_contents_steps.rb

```diff
@@ -22,35 +22,44 @@
   end
 end
 
-# When(/^(?:I|we) send (?:them their|a) record of guidance$/) do
-#   appointment_summary_page = AppointmentSummaryPage.new
-#   appointment_summary_page.load
-#   appointment_summary_page.fill_in(@appointment_summary)
-#   appointment_summary_page.submit.click
-
-#   preview_page = RecordOfGuidancePreviewPage.new
-#   preview_page.confirm.click
-# end
-
-# Then(/^the sections it includes should be \(in order\):$/) do |table|
-#   sections = table.raw.flatten
-
-#   if (index = sections.index('detail about applicable circumstances'))
-#     circumstances = []
-#     circumstances << 'continue working' if @appointment_summary.continue_working?
-#     circumstances << 'unsure' if @appointment_summary.unsure?
-#     circumstances << 'leave inheritance' if @appointment_summary.leave_inheritance?
-#     circumstances << 'wants flexibility' if @appointment_summary.wants_flexibility?
-#     circumstances << 'wants security' if @appointment_summary.wants_security?
-#     circumstances << 'wants lump sum' if @appointment_summary.wants_lump_sum?
-#     circumstances << 'poor health' if @appointment_summary.poor_health?
-
-#     sections[index] = circumstances
-#     sections.flatten!
-#   end
-
-#   expect(page).to include_output_document_sections(sections)
-# end
+When(/^(?:I|we) send (?:them their|a) record of guidance$/) do
+  appointment_summary_page = AppointmentSummaryPage.new
+  appointment_summary_page.load
+  appointment_summary_page.fill_in(@appointment_summary)
+  appointment_summary_page.submit.click
+
+  preview_page = RecordOfGuidancePreviewPage.new
+  preview_page.confirm.click
+
+  ProcessOutputDocuments.new.call
+end
+
+Then(/^the sections it includes should be \(in order\):$/) do |table|
```

Comment:
Perhaps we should remove the "in order", as it's not actually validated (or even controlled by this app) at all?

It does better express intention though.